### PR TITLE
[STAR-789] Add proposed Protection Level message

### PR DESCRIPTION
**Pull request by jbangelo**
Split this into two commits. The first contains the update to the spec, the second contains all of the regenerated code.

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/navigation.yaml [hunk 1]
```diff
@@ -1370,3 +1370,55 @@ definitions:
...
+          - z:
+              type: double
+              units: m
+              desc: ECEF Z coordinate
+          - flags:
+              type: u8
+              desc: Status flags
+              fields:
+                - 0-2:
+                    desc: TIR Level
```
Comment (lloydmaza) [inline spec/yaml/swiftnav/sbp/navigation.yaml]: Nit: expand TIR here to "Target Integrity Risk"

File: spec/yaml/swiftnav/sbp/navigation.yaml [hunk 2]
```diff
@@ -1370,3 +1370,55 @@ definitions:
...
+                    values:
+                        - 0: TIR Level 1
+                        - 1: TIR Level 2
+                        - 2: TIR Level 3
```
Comment (lloydmaza) [inline spec/yaml/swiftnav/sbp/navigation.yaml]: Maybe add the defined [TIR level specifications](https://swift-nav.atlassian.net/wiki/spaces/SWIF/pages/34242626/Glossary#Glossary-T) here.
`TIR Level 1 - 1E-5 fault/hr`
`TIR Level 2 - 1E-6 fault/hr`
`TIR Level 3 - 1E-7 fault/hr`

File: spec/yaml/swiftnav/sbp/navigation.yaml [hunk 3]
```diff
@@ -1370,3 +1370,55 @@ definitions:
...
                         - 1: Fixed RTK
+
+ - MSG_PROTECTION_LEVEL:
+    public: false
+    id: 0x0216
+    short_desc: Computed Protection Level
+    desc: |
+      This message reports the baseline heading pointing from the base station
+      to the rover relative to True North. The full GPS time is given by the
+      preceding MSG_GPS_TIME with the matching time-of-week (tow).
```
[the remark this answers is not in the document]
Comment (jbangelo) [reply spec/yaml/swiftnav/sbp/navigation.yaml]: Good catch!

File: spec/yaml/swiftnav/sbp/navigation.yaml (hunk 1 above)
Comment (jbangelo) [reply spec/yaml/swiftnav/sbp/navigation.yaml]: Done

File: spec/yaml/swiftnav/sbp/navigation.yaml (hunk 2 above)
Comment (jbangelo) [reply spec/yaml/swiftnav/sbp/navigation.yaml]: Will defer this change for later.

File: spec/yaml/swiftnav/sbp/navigation.yaml [hunk 4]
```diff
@@ -1370,3 +1370,56 @@ definitions:
...
+              fields:
+                - 0-2:
+                    desc: Target Integrity Risk (TIR) Level
+                    values:
+                        - 0: Safe state, protection level shall not be used for safety-critical application
+                        - 1: TIR Level 1
+                        - 2: TIR Level 2
+                        - 3: TIR Level 3
+                - 3:
+                    desc: PL Safe State
```
Comment (scarcanague) [inline spec/yaml/swiftnav/sbp/navigation.yaml]: So I think we dont use  this bit anymore  now that the 0 version of the 2 bits before indicates safe state

Comment (jbangelo) [reply spec/yaml/swiftnav/sbp/navigation.yaml]: Good catch. Removed it.

**Pull request merged**